.github/workflows/docker-tag.yml

```
@@ -0,0 +1,103 @@
+# This github workflow will automatically update docker image tags of rs-proxy-depl in the datakaveri/iudx-deployment repository files, whenever docker image is pushed to ghcr.io/datakaveri/rs-proxy-depl .Based on tag it will update the master/latest branch (if its 5.5.1-alpha-) or 5.5.0 stable branch (if its 5.5.0-)
+
+name: Update RS Proxy docker image tags
+
+# This trigger will run the workflow whenever a new package is published to the registry
+on:
+  registry_package:
+    types: [published]
+
+# This is needed to read the registry packages     
+permissions:
+  packages: read
+
+jobs:
+
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+       repository: datakaveri/iudx-deployment
+       # Jenkins token to perform git operations
+       token: "${{ secrets.JENKINS_UPDATE }}"
+       fetch-depth: 0
+    # This step updates the RS Server docker image tags
+    - name: Update RS Proxy docker image tags
+      env: 
+        GH_TOKEN: ${{ secrets.JENKINS_UPDATE}}
+      run: | 
+
+        # Get the latest version of 5.5.0 tags from the container registry using GitHub API
+        export newtag5_5_0=$(head -n1 <(curl -H "Accept: application/vnd.github+json" -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/orgs/datakaveri/packages/container/rs-proxy-depl/versions | jq ' .[].metadata.container.tags[0]'  | grep 5.5.0 | grep -v alpha | sed -e 's/^"//' -e 's/"$//'))
+        export newtag5_5_1=$(head -n1 <(curl -H "Accept: application/vnd.github+json" -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/orgs/datakaveri/packages/container/rs-proxy-depl/versions | jq ' .[].metadata.container.tags[0]'  | grep 5.5.1 | sed -e 's/^"//' -e 's/"$//'))
```

5.5.1-alpha is used here and not in 5.5.0. update grep in export commands accordingly. 
```suggestion
        export newtag5_5_1=$(head -n1 <(curl -H "Accept: application/vnd.github+json" -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/orgs/datakaveri/packages/container/rs-proxy-depl/versions | jq ' .[].metadata.container.tags[0]'  | grep 5.5.1-alpha | sed -e 's/^"//' -e 's/"$//'))
```